models/Element/Exception/ElementNotFoundException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @category   Pimcore
+ * @package    Element
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Model\Element\Exception;
+
+class ElementNotFoundException extends \Exception
```

Comment:
This should extend the [`RuntimeException`](https://www.php.net/manual/de/class.runtimeexception.php) class, as this only occurs on runtime.

```suggestion
class ElementNotFoundException extends \RuntimeException
```